fv/infrastructure/infrastructure.go

```diff
@@ -71,3 +73,23 @@ type DatastoreInfra interface {
 	// Stop cleans up anything necessary in preparation for the end of the test.
 	Stop()
 }
+
+// Creates a default profile that allows workloads with this profile to talk to each
+// other in the absence of any Policy.
+func CreateDefaultProfile(c client.Interface, name string, labels map[string]string, entityRuleSelector string) {
+	d := api.NewProfile()
+	d.Name = name
+	if labels != nil {
+		d.Spec.LabelsToApply = labels
+	}
+	d.Spec.Egress = []api.Rule{{Action: api.Allow}}
+
+	if len(entityRuleSelector) > 0 {
+		d.Spec.Ingress = []api.Rule{{
+			Action: api.Allow,
+			Source: api.EntityRule{Selector: entityRuleSelector},
+		}}
+	}
```

Comment:
I think we still need `d.Spec.Ingress = []api.Rule{{Action: api.Allow}}` when the selector is empty, don't we?